Tests/Builder/FormContractorTest.php

```diff
@@ -46,4 +47,68 @@ public function testGetFormBuilder()
             $this->formContractor->getFormBuilder('test', array('foo' => 'bar'))
         );
     }
+
+    public function testDefaultOptionsForSonataFormTypes()
+    {
+        $admin = $this->getMockBuilder('Sonata\AdminBundle\Admin\AdminInterface')->getMock();
```

Comment:
I would prefer `getMock` because the mock builder for an interface is useless.

> This way produces less warning noise in the test/build output.

Yes, and the mock builder will be still like this even after PHPUnit 4 support dropping because of that.

It produce deprecated warning but for now we have no choice.